src/helpers/merging.py

```diff
@@ -0,0 +1,51 @@
+from constants import OUTPUT_COLUMNS
+
+NUMERIC_COLUMNS = ['Pop Tested',
+                   'Pop Tested Positive',
+                   'Pop Tested Negative',
+                   'Pop Deaths',
+                   'Staff Tested',
+                   'Staff Tested Positive',
+                   'Staff Tested Negative',
+                   'Staff Deaths']
+TEXT_COLUMNS = ['Source',
+                'Notes']
+
+assert set(NUMERIC_COLUMNS).issubset(set(OUTPUT_COLUMNS))
+assert set(TEXT_COLUMNS).issubset(set(OUTPUT_COLUMNS))
+
+
+def get_value(row, column):
+  try:
+    return int(row[column])
+  except (KeyError, ValueError):
+    return ''
+
+
+def merge_column(column, rows_with_sources, sources_used):
+  # Use the largest count since it was probably recorded latest in the day
+  rows_with_values = list(filter(lambda row_with_source: '' != get_value(row_with_source[0], column),
+                                 rows_with_sources))
+  row = None
+  if rows_with_values:
+    row, source = max(rows_with_values,
+                      key=lambda row_with_source: get_value(row_with_source[0], column))
+
+  if row:
+    sources_used.add(source)
+
+  return get_value(row, column) if row else ''
```

Comment:
If we do change `get_value` to return None instead of an empty string, we should ensure that this particular call returns an empty string if the function returns None or have L45 below translate from None to empty string.

Reply:
Yeah, any row that doesn't have a value for the column at hand is filtered out before determining the row with the max value, so if `get_value` is invoked here, it will definitely have some value. Else, if no rows with a value for the column were provided, the empty string is returned.